lms/static/js/spec/learner_dashboard/collection_list_view_spec.js

```diff
@@ -124,6 +124,31 @@ define([
                 $cards = view.$el.find('.program-card');
                 expect($cards.length).toBe(0);
             });
+            it('should have no title when title not provided', function(){
+                var $title;
+                setFixtures('<div class="test-container"><div class="program-cards-container"></div></div>');
+                view.remove();
+                view.render();
+                expect(view).toBeDefined();
+                $title = view.$el.prev();
+                expect($title.html()).not.toBeDefined();
+            });
+            it('should display screen reader header when provided', function(){
+                var $title, titleContext = {el:'h2', title:'list start'};
+                view.remove();
+                setFixtures('<div class="test-container"><div class="program-cards-container"></div></div>');
+                programCollection = new ProgramCollection(context.programsData);
+                view = new CollectionListView({
+                    el: '.program-cards-container',
+                    childView: ProgramCardView,
+                    context: {'xseriesUrl': '/programs'},
+                    collection: programCollection,
+                    titleContext: titleContext
+                });
+                view.render();
+                $title = view.$el.prev();
```

Comment:
I don't think this test is specific enough, it should be hitting a selector that we know is the title. Not just a previous DOM element.